workflows/inngest/src/index.ts

```diff
@@ -1355,8 +1348,7 @@ export class InngestExecutionEngine extends DefaultExecutionEngine {
               steps: resume.steps.slice(1),
               stepResults: snapshot?.context as any,
               resumePayload: resume.resumePayload,
-              // @ts-ignore
-              resumePath: snapshot?.suspendedPaths?.[resume.steps?.[1]] as any,
+              resumePath: snapshot?.suspendedPaths?.[resume.steps?.[1] ?? ''] as any,
```

Comment:
We probably want to switch this to something along these lines
```suggestion
              resumePath: resume.steps?.[1] ? snapshot?.suspendedPaths?.[resume.steps?.[1]] as any : undefined,
```